Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -148,7 +163,6 @@ public void tankDrivePowerMode(double leftPower, double rightPower) {
      * @param motor used to access the motor to set and give data
      */
     private void updateMotorConfig(XCANTalon motor) {
-        motor.configEncoderCodesPerRev((int)encoderCodesProperty.get());
```

Comment:
I think @aschokking told me to remove this because they put in another implementation of this somewhere else.